configdialog/lxqtpageselectwidget.cpp

```diff
@@ -102,12 +103,14 @@ QSize PageSelectWidget::sizeHint() const
     int w = 0;
     for(int i=0; i< count(); ++i)
     {
-        QRect rect = fontMetrics().boundingRect(QRect(), Qt::AlignLeft | Qt::TextWordWrap, item(i)->text());
+        QRect rect = fontMetrics().boundingRect(QRect(), Qt::AlignHCenter | Qt::TextWordWrap, item(i)->text());
         w = qMax(w, rect.width());
     }
+    // consider the icon size too
+    w = qMax(w, QApplication::style()->pixelMetric(QStyle::PM_IconViewIconSize));
```

Comment:
> It's just in QRenderRule rule = renderRule(w, opt); before switch.

Of course... the rule is taken based on the widget. If the widget is not present (nullptr), the specific rule (if existing) will not be chosen/found.